superset/connectors/sqla/models.py

```diff
@@ -581,6 +581,7 @@ def data(self) -> Dict:
             d["main_dttm_col"] = self.main_dttm_col
             d["fetch_values_predicate"] = self.fetch_values_predicate
             d["template_params"] = self.template_params
+            d["is_sqllab_view"] = self.is_sqllab_view
```

Comment:
@graceguo-supercat does this impact the cache key in anyway? If so we should probably add a note in `UPDATING.md` to inform others.

Reply:
It adds extra attribute in the datasource object. I am not sure how cache key use datasource. Is this a relevant cache key code?
https://github.com/apache/incubator-superset/blob/a6cedaaa879348aca49a520793bb20e63d152a1f/superset/common/query_object.py#L199

Reply:
@graceguo-supercat I'm not exactly sure where this is used but I think it's unrelated to the cache. Sorry for the false alarm.